documentation/src/main/wiki/GettingStarted.md

```diff
@@ -129,12 +129,14 @@ This is the main configuration supplying config entries for each verticle starte
       ]
     }
   },
+  "number.of.instances": 2
```

Comment:
Verticles lanucher should support all possible parameters available for DeploymentOptions object.
It means:

- [ ] config section should look as follows
```json
{
  "some-verticle" : {
     "config" : {...},
     "some-param" : "aa",
     "other-param" : "bb"
 }
```
Simple on the same level as config - those params need to be the one that DeploymentOptions object support. It's covered in documentation.
- [ ] in the starter verticle, you can simply supply our verticle json into DeploymentOptions object.
Use (DeploymentOptions#fromJson() method) that will do all the job [See here how it works](https://github.com/eclipse/vert.x/blob/master/src/main/java/io/vertx/core/DeploymentOptions.java#L103).
We must not be redunant to what Vert.x giving us out of the box.